runtime/Go/antlr/utils.go

```diff
@@ -385,3 +400,36 @@ func TitleCase(str string) string {
 	//	})
 
 }
+
+// murmur hash
+const (
+	c1_32 = 0xCC9E2D51
+	c2_32 = 0x1B873593
+	n1_32 = 0xE6546B64
+)
+
+func initHash(seed int) int {
+	return seed
+}
+
+func update(h1 int, k1 int) int {
+	k1 *= c1_32
```

Comment:
@willfaught, @davesisson 
There are no current dependancies in the runtime outside of the standard library. There should be a really good reason to introduce one. Murmur hash is small, not going chancing (at least not in this universe, but lets not get meta-physical) and was copied into the Java implementation.

Looking into using something like the guava collection in the Java code is an interesting discussion. Imo, importing murmur isn't.